bgpd/bgp_vty.c

```diff
@@ -17495,7 +17495,20 @@ static struct cmd_node bgp_srv6_node = {
 
 static void community_list_vty(void);
 
-static void bgp_ac_neighbor(vector comps, struct cmd_token *token)
+static void bgp_ac_peergroup(vector comps, struct cmd_token *token)
+{
+	struct bgp *bgp;
+	struct peer_group *group;
+	struct listnode *lnbgp, *lnpeer;
+
+	for (ALL_LIST_ELEMENTS_RO(bm->bgp, lnbgp, bgp)) {
+		for (ALL_LIST_ELEMENTS_RO(bgp->group, lnpeer, group))
+			vector_set(comps,
```

Comment:
This would list all peer-groups across all instances. Is that intended? Should we not list only the peer-groups for the specific instance?

Reply:
The logic wasn't changed as it was before.

Reply:
Hmmm.. If all other auto-completions are also showing values across all instances, then I am fine with that for this PR. But I feel we should change that across all auto-completions in BGP.